lib/main.coffee

```diff
@@ -113,13 +117,18 @@ Notifications =
       unless timeSpan < @duplicateTimeDelay and notification.isEqual(@lastNotification)
         @notificationsElement.appendChild(atom.views.getView(notification).element)
         @notificationsLog?.addNotification(notification)
+        @statusBarManager?.addNotification(notification)
     else
       @notificationsElement.appendChild(atom.views.getView(notification).element)
       @notificationsLog?.addNotification(notification)
+      @statusBarManager?.addNotification(notification)
 
     notification.setDisplayed(true)
     @lastNotification = notification
 
+  statusBarService: (statusBar) ->
```

Comment:
The convention followed by other Atom packages is to name this `consumeStatusBar`.